core/model/patron.py

```diff
@@ -511,6 +511,8 @@ class Loan(Base, LoanAndHoldMixin):
     # Some distributors (e.g. Feedbooks) may have an identifier that can
     # be used to check the status of a specific Loan.
     external_identifier = Column(Unicode, unique=True, nullable=True)
+    cached_manifest = Column(Unicode, unique=True, nullable=True)
+    cached_content_type = Column(Unicode, unique=True, nullable=True)
```

Comment:
I feel like `unique` shouldn't be true here? A user should be able to have multiple loans with the same format.